pkgs/development/libraries/libcifpp/default.nix

```diff
@@ -1,19 +1,23 @@
-{ lib, stdenv, fetchFromGitHub, boost, cmake, }:
+{ lib, stdenv, fetchFromGitHub, boost, cmake, zlib, }:
 
 stdenv.mkDerivation rec {
   pname = "libcifpp";
-  version = "4.2.2";
+  version = "5.0.8";
 
   src = fetchFromGitHub {
     owner = "PDB-REDO";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0mhplcpni4p8lavrq4fz9qq8mbxhvpnlxzy55yrz8y07d76ajg6y";
+    sha256 = "sha256-KJGcopGhCWSl+ElG3BPJjBf/kvYJowOHxto6Ci1IMco=";
+    fetchSubmodules = true;
```

Comment:
```suggestion
```
there are no submodules

Reply:
I don't remember why this appeared, but thanks for finding it.